torch/testing/_internal/common_methods_invocations.py

```diff
@@ -19223,6 +19223,19 @@ def __init__(
         torch_opinfo_name="amax",
     ),
     #
+    # Linear Algebra Operators
+    #
+    PythonRefInfo(
+        "_refs.addr",
+        torch_opinfo_name="addr",
+        decorators=(
+            # No meta support for torch.tensor
+            DecorateInfo(unittest.expectedFailure, 'TestCommon', 'test_python_reference_meta_functions',),
+            # RuntimeError: no _refs support for torch.outer
+            DecorateInfo(unittest.expectedFailure, 'TestCommon', 'test_python_reference_consistency',),
```

Comment:
Eeeks! We should probably get that mode implemented before we start writing refs with torch functions if they have to skip the reference consistency test

Reply:
Well, this is what you asked for, but you are welcome to change your mind ;)

> We will have a context manager that will reroute torch.* API calls to torch._ref.* (failing if the ref doesn't exist), 

and

>> do you have a preference for two separate tests (but then I need to dupe all the xfails for the new test name I added) or one test (but then an xfail will prevent us checking that the torch.* version works)
> one test if it’s easier

My preferred resolution is to maintain only one test which does NOT test with the mode (so torch goes to torch, refs go to refs), and not test anything else. If we insist on testing the strict mode, we will need to add an extra set of tests, because any ref that calls into a torch function that doesn't have a ref will fail in strict mode. We can also test in non-strict mode, in which case you don't get any signal if all torch functions being called have refs or not.

Reply:
Yes I have a PR that will be available soon that separates the tests, though, so we can still test for consistency even without refs

Reply:
FYI https://github.com/pytorch/pytorch/pull/78026 has landed updating and separating the tests (may require a rebase)